test/helper.lua

```diff
@@ -7,5 +7,5 @@ end
 
 local checks = require('checks')
 
-local package_source = debug.getinfo(checks).source
+local package_source = debug.getinfo(checks.checks).source
```

Comment:
It would break integration tests with core. Use `if type(checks) == 'function' ...` conditions here